src/components/Global/MovieCardBasic.vue

```diff
@@ -0,0 +1,80 @@
+<script>
+import { defineComponent } from "vue";
+import ImageBasic from "@/components/Global/ImageBasic.vue";
+import MovieCategory from "@/components/Global/MovieCategory.vue";
+import MovieLength from "@/components/Global/MovieLength.vue";
+
+export default defineComponent({
+  name: "MovieCardBasic",
```

Comment:
A suggestion, but not obligatory.

`BaseMovieCard` instead of `MovieCardBasic` 

What do you think of this way of naming the components? 👀
Having the `Base` prefix in front of something would allow the components to be listed together when browsing the directories, making you immediately know they follow the same convention, plus, it follows the suggested naming scheme from the style guide:
https://vuejs.org/style-guide/rules-strongly-recommended.html#base-component-names